apps/src/templates/instructions/TeacherOnlyMarkdown.jsx

```diff
@@ -27,25 +26,25 @@ const styles = {
   }
 };
 
-const TeacherOnlyMarkdown = ({content}) => {
-  if (!content) {
-    return null;
-  }
+export default class TeacherOnlyMarkdown extends Component {
```

Comment:
Functional components cannot use the ref attribute ([source](https://react-legacy.netlify.app/docs/refs-and-the-dom.html#refs-and-functional-components)), which TopInstructions rely on, so I changed this to be a class. In addition, this required changing the way TopInstructions stored the ref so I had to update that and then updated all refs for consistency.